script/testing/reporting/parsers/oltpbench/res_parser.py

```diff
@@ -1,32 +1,38 @@
 import csv
-import json
 
-from reporting.utils import get_value_by_pattern
-from reporting.constants import LATENCY_ATTRIBUTE_MAPPING
+from ...constants import LATENCY_ATTRIBUTE_MAPPING
+from ...utils import get_value_by_pattern
 
 
 def parse_res_file(path):
-    """Read data from file ends with ".res".
-
-    Args:
-        path (str): The position of the res file.
+    """
+    Read data from an OLTPBench-generated file that ends with ".res".
 
-    Returns:
-        incremental_metrics (list, json array): The throughput at different time.
+    Parameters
+    ----------
+    path : str
+        Path to a ".res" file.
 
+    Returns
+    -------
+    incremental_metrics : [dict]
+        The throughput of the test at different times.
     """
+    gvbp = get_value_by_pattern
```

Comment:
sure, still fits in 80 columns

Reply:
Totally off-topic, but personal preference I use 120 instead of 80 cause screens have gotten bigger over the years. Just a fun fact. Though I've also started opening multiple files side by side in vscode so maybe 80 actually makes sense again.